impl/src/main/java/org/jboss/weld/logging/BootstrapLogger.java

```diff
@@ -272,4 +272,28 @@
     @Message(id = 165, value = "ProcessBeanAttributes.veto() called by {0} for {1}", format = Format.MESSAGE_FORMAT)
     void beanAttributesVetoed(Object extensionName, Object type);
 
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 166, value = "AfterTypeDiscovery.{2} modified by {0} adds {1}.", format = Format.MESSAGE_FORMAT)
+    void addClassInAfterTypeDiscovery(Object extensionName, Object type, Object types);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 167, value = "AfterTypeDiscovery.{2} modified by {0} adds {1} at position {3}.", format = Format.MESSAGE_FORMAT)
+    void addClassInAfterTypeDiscoveryToIndex(Object extensionName, Object type, Object types, int index);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 168, value = "AfterTypeDiscovery.{2} modified by {0} removes {1}.", format = Format.MESSAGE_FORMAT)
+    void removeClassInAfterTypeDiscovery(Object extensionName, Object type, Object types);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 169, value = "AfterTypeDiscovery.{2} modified by {0} sets {1}.", format = Format.MESSAGE_FORMAT)
+    void setClassInAfterTypeDiscovery(Object extensionName, Object type, Object types);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 170, value = "AfterTypeDiscovery.{1} modified by {0} removes all classes.", format = Format.MESSAGE_FORMAT)
+    void clearInAfterTypeDiscovery(Object extensionName, Object types);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 171, value = "AfterTypeDiscovery.{2} modified by {0} retains {1}.", format = Format.MESSAGE_FORMAT)
+    void retainAllInAfterTypeDiscovery(Object extensionName, Object type, Object types);
```

Comment:
I wonder whether it would be possible to use single parameterized message for all operations?